scalardb/project.clj

```diff
@@ -15,7 +15,8 @@
                                                          com.azure/azure-cosmos
                                                          io.grpc/grpc-core
                                                          com.scalar-labs/scalardb-rpc]]]}
-             :use-jars {:dependencies [[com.google.guava/guava "31.1-jre"]]
+             :use-jars {:dependencies [[com.google.guava/guava "31.1-jre"]
+                                       [org.apache.commons/commons-text "1.10.0"]]
```

Comment:
Sorry, I'm probably misunderstanding it, but I'm not sure why it is needed because the use of commons-text is closed internally in the core. Can you clarify?

Reply:
In the Jepsen test, we build a `scalardb` jar with the `master` branch (to test the latest code), and we run the Jepsen test with this jar with the `:use-jars` profile. In that case, we need to add the necessary dependencies because the `scalardb` jar is not a fat jar. That's why we need to add the `commons-text` lib in this PR.

Actually, if we use the `scalardb` SNAPSHOT version, we don't need to do that. That's future work.

Reply:
Ah, right. Thanks!